themes/custom/droopler_theme/scss/components/_d_blog_listing.scss

```diff
@@ -49,7 +49,13 @@ $d-blog-article-row-min-height: 420px !default;
 
 // Listing header (for main view and taxonomy).
 .blog-header-wrapper {
-  @include listing-header($d-blog-listing-header-background-color, $d-blog-listing-header-text-color, $d-blog-listing-header-desktop-padding, $d-blog-listing-header-mobile-padding, $d-blog-listing-header-margin);
+  @include listing-header(
+      $d-blog-listing-header-background-color,
+      $d-blog-listing-header-text-color,
+      $d-blog-listing-header-desktop-padding,
+      $d-blog-listing-header-mobile-padding,
+      $d-blog-listing-header-margin
+  );
```

Comment:
i only moved it to separate lines, to improve readability a bit... same bellow :point_down: